springdoc-openapi-tests/springdoc-openapi-javadoc-tests/src/test/java/test/org/springdoc/api/app170/Cat.java

```diff
@@ -0,0 +1,22 @@
+package test.org.springdoc.api.app170;
+
+
+import io.swagger.v3.oas.annotations.media.Schema;
+
+@Schema
+public class Cat implements Animal {
```

Comment:
Notice missing class Javadoc (and no explicit description set on `@Schema`), which should translate to null `Schema.description`